### DISCUSSION TITLE
Rocket League closing when launching with Heroic

### DISCUSSION THREAD

**Discussion by Ayanashi** [Heroic-Games-Launcher/HeroicGamesLauncher#1455] [General]
i reported the issue  [#1454](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1454) now i opening a discussion for help.. i have all version of proton and wine at last versions someone have idea why this happend?

**Comment by flavioislima** [comment]
I can run it using Wine-GE + DXVK checked on wine settings.
Try on a new prefix.

**Reply by Ayanashi** [reply]
different log with different settings..
_

> `Launch Command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/rocketleague/Binaries/Win64 DXVK_HUD=fps STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/home/lain/.config/heroic/tools/proton/Proton-GE-Proton7-20/proton' run"
> 
> System Info:
> Heroic Version: 2.3.7 Brook
> Legendary Version:  0.20.26 Dark Energy
> OS: Arch KERNEL: 5.18.2-zen1-1-zen ARCH: x64
> CPU: Intel Core™ i5-7200U @2.5 GOVERNOR: powersave
> RAM: Total: 3.71 GiB Available: 2.54 GiB
> GRAPHICS: GPU0: HD Graphics 620  VRAM: 256MB DRIVER:  GPU1: NVIDIA GeForce 930MX VRAM: 2048MB DRIVER: 515.48.07 
> PROTOCOL: x11
> 
> Game Settings: {
> 	"autoInstallDxvk": true,
> 	"autoInstallVkd3d": true,
> 	"enableEsync": false,
> 	"enableFsync": false,
> 	"maxSharpness": 2,
> 	"enableResizableBar": false,
> 	"nvidiaPrime": true,
> 	"offlineMode": false,
> 	"otherOptions": "",
> 	"savesPath": "",
> 	"showFps": true,
> 	"targetExe": "",
> 	"useGameMode": false,
> 	"wineCrossoverBottle": "Heroic",
> 	"winePrefix": "/home/lain/Games/Heroic/rocketleague/Binaries/Win64",
> 	"wineVersion": {
> 		"bin": "/home/lain/.config/heroic/tools/proton/Proton-GE-Proton7-20/proton",
> 		"name": "Proton - Proton-GE-Proton7-20",
> 		"type": "proton"
> 	},
> 	"useSteamRuntime": false
> }
> 
> Error Log:
> [cli] INFO: Logging in...
> [Core] INFO: Trying to re-use existing login session...
> [cli] INFO: Checking for updates...
> [Core] INFO: Getting authentication token...
> [cli] INFO: Launching Sugar...
> fsync: up and running.
> wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
> [S_API] SteamAPI_Init(): SteamAPI_IsSteamRunning() did not locate a running instance of Steam.
> [S_API FAIL] SteamAPI_Init() failed
> 
> Game Log:
> Warning, Failed to compile Material EngineMaterials.FogVolumeMaterial for platform PC-D3D-SM5, Default Material will be used in game.
> Warning, Failed to compile Material EngineResources.M_StreamingPause for platform PC-D3D-SM5, Default Material will be used in game.
> Warning, Failed to compile Material NodeBuddies.Materials.NodeBuddy_Red1 for platform PC-D3D-SM5, Default Material will be used in game.
> Warning, Failed to compile Material NodeBuddies.Materials.NodeBuddy_Target for platform PC-D3D-SM5, Default Material will be used in game.
> Warning, Failed to compile Material NodeBuddies.Materials.NodeBuddy_Text1 for platform PC-D3D-SM5, Default Material will be used in game.
> Warning, Failed to compile Material NodeBuddies.Materials.NodeBuddy_White1 for platform PC-D3D-SM5, Default Material will be used in game.
> `

_

**Reply by Ayanashi** [reply]
same result

**Comment by Ayanashi** [comment]
**NEW LOG WITH WINE-PROTON-GE**
_`Launch Command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia WINEPREFIX=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --wine /home/lain/.config/heroic/tools/wine/Wine-GE-Proton7-16/bin/wine --wine-prefix /home/lain/Games/Heroic/Prefixes/rocketleague

System Info:
Heroic Version: 2.3.8 Brook
Legendary Version:  0.20.26 Dark Energy
OS: Arch KERNEL: 5.18.3-zen1-1-zen ARCH: x64
CPU: Intel Core™ i5-7200U @2.5 GOVERNOR: powersave
RAM: Total: 3.71 GiB Available: 2.77 GiB
GRAPHICS: GPU0: HD Graphics 620  VRAM: 256MB DRIVER:  GPU1: NVIDIA GeForce 930MX VRAM: 2048MB DRIVER: 515.48.07 
PROTOCOL: x11

Game Settings: {
	"autoInstallDxvk": true,
	"autoInstallVkd3d": false,
	"enableEsync": false,
	"enableFsync": false,
	"maxSharpness": 2,
	"enableResizableBar": false,
	"nvidiaPrime": true,
	"offlineMode": false,
	"otherOptions": "",
	"savesPath": "",
	"showFps": true,
	"targetExe": "",
	"useGameMode": false,
	"wineCrossoverBottle": "Heroic",
	"winePrefix": "/home/lain/Games/Heroic/Prefixes/rocketleague",
	"wineVersion": {
		"bin": "/home/lain/.config/heroic/tools/wine/Wine-GE-Proton7-16/bin/wine",
		"name": "Wine - Wine-GE-Proton7-16",
		"type": "wine",
		"wineserver": "/home/lain/.config/heroic/tools/wine/Wine-GE-Proton7-16/bin/wineserver",
		"wineboot": "/home/lain/.config/heroic/tools/wine/Wine-GE-Proton7-16/bin/wineboot"
	},
	"useSteamRuntime": false
}

Error Log:
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching Sugar...
wineserver: using server-side synchronization.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
002c:fixme:winediag:LdrInitializeThunk wine-staging 7.0 is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please mention your exact version when filing bug reports on winehq.org.
0070:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\wineusb": c0000142
003c:fixme:service:scmdatabase_autostart_services Auto-start service L"wineusb" failed to start: 1114
0098:err:xrandr:xrandr14_get_adapters Failed to get adapters
0128:fixme:kernelbase:AppPolicyGetThreadInitializationType FFFFFFFFFFFFFFFA, 0000000001E5FF80
0124:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
0124:err:wldap32:DllMain No libldap support, expect problems
0124:err:kerberos:kerberos_LsaApInitializePackage no Kerberos support, expect problems
info:  Game: RocketLeague.exe
info:  DXVK: v1.10.1
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  NVIDIA GeForce 930MX:
info:    Driver: 515.48.7
info:    Vulkan: 1.3.205
info:    Memory Heap[0]: 
info:      Size: 2048 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:    Memory Heap[1]:
info:      Size: 2852 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[8]: Property Flags = 0x6
info:      Memory Type[9]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 246 MiB
info:      Flags: 0x1
info:      Memory Type[10]: Property Flags = 0x7
info:  DXGI: NvAPI workaround enabled, reporting AMD GPU
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_0
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_0
info:  Device properties:
info:    Device name:     : NVIDIA GeForce 930MX
info:    Driver version   : 515.48.7
info:  Enabled device extensions:
info:    VK_EXT_4444_formats
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_extended_dynamic_state
info:    VK_EXT_host_query_reset
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:
VK_KHR_driver_properties
info:    VK_KHR_external_memory_win32
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_float_controls
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 0
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 0
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 1
info:    formatA4B4G4R4                         : 1
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 1
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 0
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  VK_KHR_buffer_device_address
info:    bufferDeviceAddress                    : 0
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 2 compiler threads
info:  DXGI: NvAPI workaround enabled, reporting AMD GPU
info:  Game: RocketLeague.exe
info:  DXVK: v1.10.1
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  NVIDIA GeForce 930MX:
info:    Driver: 515.48.7
info:    Vulkan: 1.3.205
info:    Memory Heap[0]: 
info:      Size: 2048 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 2852 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[8]: Property Flags = 0x6
info:      Memory Type[9]: Property Flags = 0xe
info:    Memory Heap[2]: 
info:      Size: 246 MiB
info:      Flags: 0x1
info:      Memory Type[10]: Property Flags = 0x7

0124:fixme:nvapi:unimplemented_stub function 0xbe7692ec is unimplemented!
0124:fixme:nvapi:unimplemented_stub function 0x6c2d048c is unimplemented!
015c:fixme:imm:ImeSetActiveContext (0x4803d0, 0): stub
015c:fixme:imm:ImmReleaseContext (0000000000020030, 00000000004803D0): stub
0124:fixme:imm:ImeSetActiveContext (0xbb80c90, 1): stub
0124:fixme:imm:ImmReleaseContext (000000000001005C, 000000000BB80C90): stub
0124:fixme:faultrep:ReportFault 00000000004EDFF0 0x0 stub
`_

**Comment by Ayanashi** [comment]
_`(14:59:24) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(14:59:25) INFO:    [Legendary]:       Legendary location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(14:59:25) INFO:    [Gog]:             GOGDL location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(14:59:25) INFO:    [Backend]:         

Heroic Version: 2.3.8 Brook
Legendary Version:  0.20.26 Dark Energy
OS: Arch KERNEL: 5.18.3-zen1-1-zen ARCH: x64
CPU: Intel Core™ i5-7200U @2.5 GOVERNOR: powersave
RAM: Total: 3.71 GiB Available: 2.74 GiB
GRAPHICS: GPU0: HD Graphics 620  VRAM: 256MB DRIVER:  GPU1: NVIDIA GeForce 930MX VRAM: 2048MB DRIVER: 515.48.07 
PROTOCOL: x11

(14:59:28) WARNING: [Backend]:         Protocol already registered.
(14:59:30) INFO:    [Backend]:         Downloaded Winetricks
(14:59:30) INFO:    [Frontend]:        Refreshing Library
(14:59:30) INFO:    [Legendary]:       Refreshing library...
(14:59:30) INFO:    [Legendary]:       Refreshing Epic Games...
(14:59:32) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list
(14:59:34) INFO:    [Legendary]:       Updating game list
(14:59:34) INFO:    [Legendary]:       Game List Updated
(14:59:34) INFO:    [Legendary]:       Checking for game updates: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(14:59:34) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary list-installed --check-updates --tsv
(14:59:35) INFO:    [Legendary]:       Found 0 game(s) to update
(14:59:35) INFO:    [Gog]:             Found 0 game(s) to update
(15:02:06) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(15:02:06) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(15:02:06) WARNING: [Legendary]:       No Steam runtime found
(15:02:06) INFO:    [Legendary]:       Launching Rocket League®: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/usr/share/steam/compatibilitytools.d/proton-ge-custom/proton' run"
(15:02:06) DEBUG:   [Legendary]:       Running Legendary command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/usr/share/steam/compatibilitytools.d/proton-ge-custom/proton' run"
(15:04:53) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(15:04:57) INFO:    [Backend]:         Log was written to /home/lain/.config/heroic/GamesConfig/Sugar-lastPlay.log
(15:05:02) INFO:    [Frontend]:        Refreshing Library
(15:05:02) INFO:    [Legendary]:       Refreshing library...
(15:05:05) INFO:    [Legendary]:       Updating game list
(15:05:05) INFO:    [Legendary]:       Game List Updated
(15:09:06) INFO:    [Frontend]:        Refreshing wine downloader releases
(15:09:06) INFO:    [WineDownloader]:  Updating wine versions info
(15:09:06) INFO:    [WineDownloader]:  Fetching upstream information...
(15:09:08) INFO:    [WineDownloader]:  wine versions updated
(15:15:30) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(15:15:31) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(15:15:31) WARNING: [Legendary]:       No Steam runtime found
(15:15:31) INFO:    [Legendary]:       Launching Rocket League®: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/usr/share/steam/compatibilitytools.d/proton-ge-custom/proton' run"
(15:15:31) DEBUG:   [Legendary]:       Running Legendary command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/usr/share/steam/compatibilitytools.d/proton-ge-custom/proton' run"
(15:18:54) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(15:19:01) INFO:    [Backend]:         Log was written to /home/lain/.config/heroic/GamesConfig/Sugar-lastPlay.log
(15:19:09) INFO:    [Frontend]:        Refreshing Library
(15:19:09) INFO:    [Legendary]:       Refreshing library...
(15:19:13) INFO:    [Legendary]:       Updating game list
(15:19:13) INFO:    [Legendary]:       Game List Updated
(15:21:50) INFO:    [Backend]:         Launching Rocket League® (Sugar)
(15:21:50) WARNING: [Backend]:         You are using Proton, this can lead to some bugs. Please do not open issues with bugs related to games
(15:21:50) WARNING: [Legendary]:       No Steam runtime found
(15:21:50) INFO:    [Legendary]:       Launching Rocket League®: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/home/lain/.config/heroic/tools/proton/Proton-GE-Proton7-20/proton' run"
(15:21:50) DEBUG:   [Legendary]:       Running Legendary command: DRI_PRIME=1 __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/lain/.steam/steam STEAM_COMPAT_DATA_PATH=/home/lain/Games/Heroic/Prefixes/rocketleague DXVK_HUD=fps WINEFSYNC=1 STEAM_COMPAT_APP_ID=0 SteamAppId=0 SteamGameId=heroic-rocketleague PROTON_LOG_DIR=/home/lain /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --no-wine --wrapper "'/home/lain/.config/heroic/tools/proton/Proton-GE-Proton7-20/proton' run"
(15:26:43) DEBUG:   [Legendary]:       Running Legendary command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary --version
(15:26:49) INFO:    [Backend]:         Log was written to /home/lain/.config/heroic/GamesConfig/Sugar-lastPlay.log
(15:27:01) INFO:    [Frontend]:        Refreshing Library
(15:27:01) INFO:    [Legendary]:       Refreshing library...
(15:27:03) INFO:    [Legendary]:       Updating game list
(15:27:03) INFO:    [Legendary]:       Game List Updated
`_

**Comment by Ayanashi** [comment]
nothing changed after some weeks i change some settings and paths not work
